astropy/wcs/tests/test_tab.py

```diff
@@ -61,11 +61,17 @@ def test_mixed_celest_and_1d_tab_roundtrip():
     assert np.allclose(pts, w.wcs_world2pix(w.wcs_pix2world(pts, 0), 0))
 
 
+@pytest.mark.skipif(
+    _WCSLIB_VER < Version('7.8'),
```

Comment:
Do you think applying this `skipif` to #13090 would make the backport of #13063 possible?

Reply:
Yes

Reply:
But maybe this `skipif` should be backported together with changing of the `ctype` from `TIME` to `FREQ` because there were issues with `TIME` in version 7.8 that were fixed in 7.9 and so by modifying the test to use `FREQ` we can lower `skipif` to version 7.8. Otherwise, if not changing the unit test in the backport, the `skipif` should be set to WCSLIB version `7.9`.

Reply:
When you get around to opening PR for this backport, please let me know if it supersedes #13090 or not. Thanks!

Reply:
@pllim https://github.com/astropy/astropy/pull/13120 fixes #13063 unit test when run with old stand-alone WCSLIB versions. It does not supersede #13090.